lib/model.d.ts

```diff
@@ -325,7 +325,7 @@ export interface IncludeOptions {
   /**
    * Load further nested related models
    */
-  include?: Includeable[];
+  include?: Includeable[] | IncludeAll[];
```

Comment:
Ah, okay. Yeah, that changes the requirements a bit...
`export type Includeable = Model | Association | IncludeOptions | { all: true };` with `include?: Includeable[];` would work then, but it wouldn't fulfill your requirement either.

Ideally, we would use your suggestion from above: `export type Includeables = Includeable[] | [{ all: true }];`, but that does not seem to work (as mentioned).

Should we settle on `export type Includeable = Model | Association | IncludeOptions | { all: true };`, maybe including additional comments about only one element being allowed in the array if `{ all: true }` is included?
Or can you think of any other way to achieve this?